SlackHistoryViewer.Helpers/Requests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Net.Http;
+using SlackHistoryViewer.Configuration;
+
+namespace RemoteMessages
+{
+    public class Requests
```

Comment:
i don't think that this class has to exists in Helpers, it has more sense in the RemoteMessages  assembly